sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py

```diff
@@ -132,7 +119,7 @@ def start_timer_and_subscriber(self):
         if not self._subscriber:
             self._subscriber = rospy.Subscriber('/{}/tactile'.format(self._side), ShadowPST,
                                                 self._tactile_data_callback)
-            self._timer.timeout.connect(self.timerEvent)
+            self._timer.timeout.connect(self.timer_event)
```

Comment:
TimerEvent is a method that is part of the ancestor of the object https://doc.qt.io/qt-5/qobject.html#timerEvent, so even though this version would still work, it would consume a bit more processor as timerEvent would be called once for nothing and the other callback will be called at the same time.

Reply:
So it would be better to keep it as TimerEvent